src/main/webapp/WEB-INF/tags/instructor/studentList.tag

```diff
@@ -0,0 +1,178 @@
+<%@ tag description="instructorSearch / instructorStudentList - Student List" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="courseId" required="true" %>
+<%@ attribute name="courseIndex" required="true" %>
+<%@ attribute name="hasSection" required="true" %>
+<%@ attribute name="sections" type="java.util.Collection" required="true" %>
+<%@ attribute name="fromStudentListPage" %>
+<table class="table table-responsive table-striped table-bordered margin-0">
+    <c:choose>
+        <c:when test="${not empty sections}">
+            <thead class="background-color-medium-gray text-color-gray font-weight-normal">
+                <tr id="resultsHeader-${courseIndex}">
+                    <th>Photo</th>
+                    <th id="button_sortsection-${courseIndex}" class="button-sort-none<c:if test="${not hasSection}"> hidden</c:if>" onclick="toggleSort(this,2)">
```

Comment:
I personally believe it's best to use `<c:if>` if there's only one option and `${ ? : }` if there are two.

We don't have any standard regarding this, though, so both are fine.